internal/server/iigointernal/orchestration.go

```diff
@@ -112,7 +112,10 @@ func RunIIGO(g *gamestate.GameState, clientMap *map[shared.ClientID]baseclient.C
 	executiveBranch.sendSpeakerSalary(&legislativeBranch)
 
 	// 1 Judge action - inspect history
-	_, historyInspected := judicialBranch.inspectHistory(g.IIGOHistory)
+	historyInspected := true
+	if g.Turn > 0 {
+		_, historyInspected = judicialBranch.inspectHistory(g.IIGOHistory[g.Turn-1])
```

Comment:
what is discarded btw?

Reply:
The evaluations, I think it will be used in the orchestration changes that are upcoming (@Spruudis), if not we will remove from function return then

Reply:
leave as such :) good to go. thanks for this.